Program.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Математические_формулы.Методы
+{
+    class Program
+    {
+        static void Main(string[] args)
+  
+        {
+            Console.WriteLine("Введите произвольную переменную");

```

Comment:
Сообщение следует формулировать более точно
```suggestion
            Console.WriteLine("Введите произвольное число");
```